qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -539,7 +540,18 @@ def find_at_pos(self, pos, callback):
 
 class WebEngineTab(browsertab.AbstractTab):
 
-    """A QtWebEngine tab in the browser."""
+    """A QtWebEngine tab in the browser.
+
+    Signals:
+        loadFinishedFake:
+            Used in place of unreliable loadFinished
+        loadProgressFake:
+            Used in place of loadProgress
+    """
+
+    #WORKAROUND for https://bugreports.qt.io/browse/QTBUG-65223
+    loadFinishedFake = pyqtSignal(bool)
+    loadProgressFake = pyqtSignal(int)
```

Comment:
qutebrowser uses Python naming conventions whenever possible (i.e. when not overriding a Qt method), so this should be `load_finished_fake` and `load_progress_fake`, even if that's a bit inconsistent with the real names. However, those also can be private as we only use them in this class, so it should be `_load_*_fake` with a leading `_`.